src/map/utils/petutils.cpp

```diff
@@ -1712,11 +1740,24 @@ namespace petutils
                 PPet->addModifier(Mod::MDEF, PChar->PMeritPoints->GetMeritValue(MERIT_FINE_TUNING, PChar));
             }
         }
+        else if (PPet->getPetType() == PETTYPE_LUOPAN && PMaster->objtype == TYPE_PC)
+        {
+            PPet->SetMLevel(PMaster->GetMLevel());
+
+            PPet->health.maxhp = PMaster->GetMaxHP();
```

Comment:
Is this actually the case (that a luopan's HP is the master's max HP), or was this done just for initial prototyping?

I'm not going to require a change (now) if not, I'd just be a surprised if that _is_ how it worked.

Reply:
HP was actually 1500 max at 99, luopan is capped at 99 also, you can increase the max hp but only through equipment.